lib/dependency-checker.js

```diff
@@ -16,9 +16,12 @@ function isUnsatisfied(pkg) {
 function EmberCLIDependencyChecker(project) {
   this.name    = 'ember-cli-dependency-checker';
   this.project = project;
-  this.checkDependencies();
 }
 
+EmberCLIDependencyChecker.prototype.included = function(/*app*/) {
+  this.checkDependencies();
```

Comment:
It is called twice, but there is a guard to prevent double warnings (because the constructor setup would have had the same problem.

https://github.com/quaertym/ember-cli-dependency-checker/blob/96b9b2a3599a013b152cff6fb547272fc634f509/lib/dependency-checker.js#L24-L26